src/components/CalendarView/__tests__/__snapshots__/index.spec.js.snap

```diff
@@ -312,20 +312,40 @@ exports[`CalendarView should render without errors 1`] = `
   margin-left: auto;
 }
 
-.c9 {
+.c10 {
   display: block;
   padding: 8px 8px 0;
   border: 0;
   color: #999999;
 }
 
-.c8:hover .c11 {
+.c9:hover .c11 {
   color: #026cdf;
   fill: currentColor;
 }
 
+@media screen and (min-width:480px) {
+
+}
+
+@media screen and (min-width:480px) {
+
+}
+
+@media screen and (min-width:480px) {
+
+}
+
+@media screen and (min-width:480px) {
+
+}
+
+@media screen and (min-width:480px) {
+
+}
```

Comment:
Please investigate why there render empty

Reply:
Errors in style property definition. Described above.

Reply:
Fixed.